packages/itwinui-css/src/tabs/tabs.scss

```diff
@@ -95,13 +102,16 @@
 }
 
 @mixin iui-tabs-horizontal {
-  flex-direction: column;
+  grid-template-areas: 'tabs tabs-actions' 'tabs-content tabs-content';
+  grid-template-columns: 1fr auto;
 
   .iui-tabs {
     display: flex;
   }
 
   .iui-tab {
+    min-height: calc(var(--iui-size-l) * 2);
```

Comment:
continuing conversation from https://github.com/iTwin/iTwinUI/pull/860#discussion_r1064638779

we can define the 44px at top

```
$tab-borderless-min-height: calc(var(--iui-size-s) * 3 + var(--iui-size-xs)); // 44px
```

and then use it here
```suggestion
    min-height: $tab-borderless-min-height;
```

Reply:
Implemented the above-mentioned change (https://github.com/iTwin/iTwinUI/pull/860/commits/0dcbbaf20de8ba8cd835a8a1774c54187384f627)

Reply:
any reason to define these in the nested selector instead of at the top of the file?

also, there is an `$iui-actions-gap` at the top, which now looks a bit confusing next `$tab-actions-gap`.

Reply:
> any reason to define these in the nested selector instead of at the top of the file?

I thought since `$tabs-actions-gap` is used just once, it would be better to keep it closer to its usage, instead of the top of the file.

> also, there is an `$iui-actions-gap` at the top, which now looks a bit confusing next `$tab-actions-gap`.

I went through all the used variable names, made them more self-explanatory and distinct from each other, and added them to the top of the file with comments. (https://github.com/iTwin/iTwinUI/pull/860/commits/7b56399c28519864018bfe2107330eca4cd4cd5b)

Reply:
As discussed, here are the latest spacing values for horizontal tabs:
* Default horizontal tabs:
  * Min height: `40px` (unchanged, since we want to minimize height [#860 (comment)](https://github.com/iTwin/iTwinUI/pull/860#discussion_r1062889038))
  * Min bottom spacing: `--iui-size-2xs` (`4px`)
* Borderless horizontal tabs:
  * Min height: `44px`
  * Min bottom spacing: `--iui-size-xs` (`8px`)
* Pill horizontal tabs:
  * Min height: `40px` (unchanged)
  * Min bottom spacing: N/A (Pills don't support adding actions)

<details>
<summary>See screenshots</summary>

![image](https://user-images.githubusercontent.com/45748283/211408504-9976f0cd-6853-45ab-8875-9b4865a33e88.png)
![image](https://user-images.githubusercontent.com/45748283/211408521-1afe7e97-1545-451d-84e8-7c3d891735eb.png)
![image](https://user-images.githubusercontent.com/45748283/211408528-c17f8969-da97-4976-ba5a-ac7e0df98da0.png)
![image](https://user-images.githubusercontent.com/45748283/211408538-e900ce78-e1fa-404b-9aaf-07dde5e1fd0a.png)

</details>